src/client/components/Threads.tsx

```diff
@@ -311,7 +312,16 @@ export const LoadMoreThreadsContext = createContext<{
 });
 
 function ThreadsScrollContainer(props: betaV2.ScrollContainerProps) {
-  const { fetchMore, loading, hasMore } = useContext(LoadMoreThreadsContext);
+  const ref = useRef<HTMLDivElement>(null);
+
+  const { threads, fetchMore, loading, hasMore } = useContext(
+    LoadMoreThreadsContext,
+  );
+
+  const [prevLatestMessageID, setPrevLatestMessageID] = useState<string | null>(
+    null,
+  );
```

Comment:
Should this be a ref so that it doesn't cause a rerender -- I don't think anything on the render path uses it?